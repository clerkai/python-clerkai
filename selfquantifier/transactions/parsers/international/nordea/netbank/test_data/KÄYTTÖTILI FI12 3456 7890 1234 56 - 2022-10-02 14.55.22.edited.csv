Bokföringsdag;Belopp;Avsändare;Mottagare;Namn;Rubrik;Referensnummer;Valuta
Reserverat;-300,00;FI12 3456 7890 1234 56;;;Revolut**1234*;;EUR
2022/09/28;-7,00;FI12 3456 7890 1234 56;;;Nordea Bank Abp;;EUR
2022/09/26;-2,00;FI12 3456 7890 1234 56;;;PAYPAL *FOO;;EUR
