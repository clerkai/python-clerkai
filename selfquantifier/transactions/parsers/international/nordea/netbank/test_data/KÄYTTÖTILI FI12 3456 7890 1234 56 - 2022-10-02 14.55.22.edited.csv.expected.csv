Real Date,Bank Date,Payer,Payee,Bank Message,Amount,Balance,Currency,Original data,Raw Real Date,Raw Bank Date,Raw Payer,Raw Payee,Raw Bank Message,Raw Amount,Raw Balance,Raw Currency
,2022-09-28,FI12 3456 7890 1234 56,,Nordea Bank Abp,-7.00,,EUR,"{'Bokföringsdag': '2022/09/28', 'Belopp': -7.0, 'Avsändare': 'FI12 3456 7890 1234 56', 'Mottagare': nan, 'Namn': nan, 'Rubrik': 'Nordea Bank Abp', 'Saldo': None, 'Referensnummer': nan, 'Valuta': 'EUR'}",,2022/09/28,FI12 3456 7890 1234 56,,Nordea Bank Abp,-7.0,,EUR
,2022-09-26,FI12 3456 7890 1234 56,,PAYPAL *FOO,-2.00,,EUR,"{'Bokföringsdag': '2022/09/26', 'Belopp': -2.0, 'Avsändare': 'FI12 3456 7890 1234 56', 'Mottagare': nan, 'Namn': nan, 'Rubrik': 'PAYPAL *FOO', 'Saldo': None, 'Referensnummer': nan, 'Valuta': 'EUR'}",,2022/09/26,FI12 3456 7890 1234 56,,PAYPAL *FOO,-2.0,,EUR
